cirq-web/cirq_ts/src/bloch_sphere/components/enums.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Enum to used for Meridians to determine the
+ * direction they should face and what type
+ * of meridian they are
+ */
+export enum Orientation {
+  HORIZONTAL_CHORD,
```

Comment:
I think we won't ever use the `HORIZONTAL` one...we could just rename `HORIZONTAL_CHORD` to `HORIZONTAL` and then remove the other.

Reply:
Rip horizontal meridians..I thought they could be a nice bonus feature sometime. I'll take them out and all the code for it

Reply:
Principle: [YAGNI ](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)